interfaces/IAirplay.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "Module.h"
+
+namespace WPEFramework {
+    namespace Exchange {
+
+        struct EXTERNAL IAirplay : virtual public Core::IUnknown {
+            enum { ID = ID_AIRPLAY};
+
+            virtual ~IAirplay() { }
+
+            enum state : uint16_t {
+                PLAYING = 0x0001,
+                STOPPED = 0x0002,
+                SUSPENDING = 0x0004
+            };
+
+            struct EXTERNAL INotification : virtual public Core::IUnknown {
+                  enum { ID = ID_AIRPLAY_NOTIFICATION };
+                  virtual void StateChange(const IAirplay::state state) = 0;
+                  virtual void Exit(const uint32_t exitCode) = 0;
+                  virtual void NotifyClient(const string& eventName) = 0;
+            };
+
+            virtual void Register(IAirplay::INotification* airplay) = 0;
+            virtual void Unregister(IAirplay::INotification* airplay) = 0;
+            virtual void FactoryReset() = 0;
+            virtual void SetVisible(bool visibility) = 0;
+            virtual void SystemCommand(const string& command) = 0;
+            virtual uint32_t startAirplayApp(const string& parametersJson) = 0;
```

Comment:
Naming incorrectly, all Interface methods are using PascalCase. Also there some repetition in here. This is Part of the AirPlay interface so wht Repeat AirPlay in the name here. I guess a Proper name here would be *uint32_t StartApp (const string& parameters) = 0;*

Reply:
I agree with fixing as PascalCase. I would like to fix this as StartAirPlayApp. This name is used in airplay-daemon to start Airplay application and just to sync with that name AirPlay was kept as it is. I hope it is fine.

Reply:
Actually I checked other .json files. Looks like it is preferred way to keep short names. I will keep "StartApp" and "StopApp". Thanks.